### ISSUE TITLE
Moving a point to a vertex lists the point as deleted

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/iD#7373]
When I move a standalone point to the edge of a building, iD creates a new node and removes the older one with its history.
I think it would be better to keep the history. JOSM saves history in such cases.

I don't remember how is it with extracting a point.

**Comment by maro-21** [comment]
Oops, sorry, in fact, the point is not deleted. It's only a message in the editor.
Could we change the message from "Deleted node" to "Modified node"?

**Comment by quincylvania** [comment]
@maro-21 Hmm where in the UI are you seeing that message? I'm seeing this after doing that action.

<img width="256" alt="Screen Shot 2020-02-24 at 10 12 09 AM" src="https://user-images.githubusercontent.com/2046746/75179107-3800c080-56ee-11ea-9e9e-aee6ebec7e77.png">

**Comment by maro-21** [comment]
After clicking "Save"

**Comment by quincylvania** [comment]
I guess this is misleading when all the mapper did was drag a point onto a vertex…

<img width="333" alt="Screen Shot 2020-02-25 at 12 37 06 PM" src="https://user-images.githubusercontent.com/2046746/75285495-964fa100-57cb-11ea-844d-1c6d4be21acf.png">

**Comment by maro-21** [comment]
Yes, this one! I thought the node will be deleted with its history, but in fact it wasn't.
